xcode/coding/coding.xcodeproj/project.pbxproj

```diff
@@ -838,14 +838,15 @@
 				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
 				GCC_PREPROCESSOR_DEFINITIONS = (
 					"$(inherited)",
-					"U_DISABLE_RENAMING=1",
+					UCONFIG_USE_LOCAL,
```

Comment:
What is it?

Reply:
Forces include of our own configuration file `3party/icu/uconfig_local.h`
See https://unicode-org.github.io/icu-docs/apidoc/released/icu4c/uconfig_8h_source.html#l00054